### PR TITLE
docs: update `devServer.https` example

### PR DESCRIPTION
**Pull request by iStuffs**
update example for documentation https key and cert only needs paths to certificate files

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]
Thanks!

File: packages/schema/src/config/dev.ts [hunk 1]
```diff
@@ -7,12 +7,11 @@ export default defineUntypedSchema({
      *
      * @example
      * ```
-     * import { fileURLToPath } from 'node:url'
      * export default {
```
Comment (danielroe) [inline packages/schema/src/config/dev.ts:10]:
```suggestion
     * export default defineNuxtConfig({
```

File: packages/schema/src/config/dev.ts [hunk 2]
```diff
@@ -7,12 +7,11 @@ export default defineUntypedSchema({
...
      *   server: {
      *     https: {
-     *       key: fs.readFileSync(fileURLToPath(new URL('./server.key', import.meta.url))),
-     *       cert: fs.readFileSync(fileURLToPath(new URL('./server.crt', import.meta.url)))
+     *       key: './server.key',
+     *       cert: './server.crt'
      *     }
      *   }
      * }
```
Comment (danielroe) [inline packages/schema/src/config/dev.ts:17]:
```suggestion
     * })
```

File: packages/schema/src/config/dev.ts [hunk 3]
```diff
@@ -7,12 +7,11 @@ export default defineUntypedSchema({
      *
      * @example
      * ```
-     * import { fileURLToPath } from 'node:url'
      * export default {
      *   server: {
```
Comment (danielroe) [inline packages/schema/src/config/dev.ts:11]:
```suggestion
     *   devServer: {
```

**Comment by dpi** [comment]
Thanks! this saved me a bunch of time messing around....

Until new docs are deployed, in case anyone is Googling:

> s.startsWith is not a function

**Pull request merged**